core/code/entity_decode.js

```diff
@@ -111,9 +109,23 @@ function detailsPortalData(a) {
   }
 };
 
-function extendedPortalData(a, required) {
+// on getEntities, intel send the history unless the user has never interacted with the portal
+// raw data is either a summary or an extended data with null details
+// when it is a summary, history is forced to 0
+function extendedPortalData(a) {
+  // default value on entities from getEntities
+  if (a.length == SUMMARY_PORTAL_DATA_LENGTH)
+    return {
+      history: parseHistoryDetail(0), // never interacted with this portal
+    }
+  // default value if no extended portal data
+  if (a.length < EXTENDED_PORTAL_DATA_LENGTH)
+    return {
+      history: null, // no data
```

Comment:
But Extended data always contain history info, so `null` here means that the portal never was interacted.

Reply:
If we get neither a summary or extended data, that's the result of a details requests that doesn't answer history even if the portal was interacted with.